content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -2,34 +2,34 @@
 After-Sales services
 ====================
 
-*After-Sales* services can be configured in the *Helpdesk* application for individual *teams*. Once
+*After-Sales* services can be configured in the *Helpdesk* application for individual teams. Once
 enabled, users can issue refunds, process returns, generate coupons, and/or schedule repair and
 field service interventions directly from a ticket.
 
 Set up the after-sales services
 ===============================
 
-Start by enabling the after-sales services on specific *Helpdesk* team(s), by going to
-:menuselection:`Helpdesk --> Configuration --> Teams` and selecting which teams(s) these services
-should be active on. Then, scroll to the :guilabel:`After-Sales` section on the team's settings
-page, and choose which of the following options to enable:
+Start by enabling the after-sales services on specific a *Helpdesk* team, by going to
+:menuselection:`Helpdesk app --> Configuration --> Helpdesk Teams` and selecting which teams these
+services should be active on. Then, scroll to the :guilabel:`After-Sales` section on the team's
+settings page, and choose which of the following options to enable:
 
-- :guilabel:`Refunds`: issues credit notes to refund a customer, or adjust the remaining amount due
-- :guilabel:`Coupons`: offers discounts and free products through an existing coupon program
-- :guilabel:`Returns`: initiates a product return from a customer through a reverse transfer
-- :guilabel:`Repairs`: creates repair orders for broken or faulty products
-- :guilabel:`Field Service`: plans onsite intervention through the *Field Service* application
+- :guilabel:`Refunds`: issues credit notes to refund a customer, or adjust the remaining amount due.
+- :guilabel:`Coupons`: offers discounts and free products through an existing coupon program.
```

Comment:
is it existing, or can it be created after? Is one of the following more accurate? Also, not sure if you want to go into the details for making a coupon program. 

offers discounts and free products through a coupon program.
or
offers discounts and free products through a coupon program, which needs to be created after enabling this option.

Reply:
Ahh, I need to read the whole doc FIRST before starting my review =D . You can link to the coupon section here. There's a lot of opportunities to link to sections that are related to what is being said through the doc, if you want.